packages/charts/react-charting/src/utilities/utilities.ts

```diff
@@ -347,7 +347,13 @@ export function prepareDatapoints(
     : (maxVal - minVal) / splitInto >= 1
     ? Math.ceil((maxVal - minVal) / splitInto)
     : (maxVal - minVal) / splitInto;
-  const dataPointsArray: number[] = [minVal, minVal + val];
+  const dataPointsArray: number[] = [minVal < 0 && maxVal >= 0 ? 0 : minVal];
+  if (minVal < 0 && maxVal >= 0) {
```

Comment:
what happens if both minVal and maxVal < 0

Reply:
For (minVal<0 and maxVal<0) and (minVal>0 and maxVal>0)
the default while loop will run

Reply:
Line 357